### PR TITLE
Expose `peerId`

### PR DESCRIPTION
**Pull request by mattcollier**
With this, `peerId` is exposed via the HTTP like this:
```
{
  "@context": "https://w3id.org/webledger/v1",
  "id": "https://bedrock.localhost:18443/ledger-agents/b75ce2a9-bcf0-48e7-b2b0-ed615f42f431",
  "owner": "did:v1:0a02328e-ba9d-43f8-830c-f05105495d66",
  "peerId": "https://bedrock.localhost:18443/consensus/continuity2017/voters/z6Mknf211dimgKwsD7HEagska5XZS17LNFGMccP5KsMEfFWU",
  "public": false,
  "service": {
    "ledgerAgentStatusService": "https://bedrock.localhost:18443/ledger-agents/b75ce2a9-bcf0-48e7-b2b0-ed615f42f431",
    "ledgerConfigService": "https://bedrock.localhost:18443/ledger-agents/b75ce2a9-bcf0-48e7-b2b0-ed615f42f431/config",
    "ledgerOperationService": "https://bedrock.localhost:18443/ledger-agents/b75ce2a9-bcf0-48e7-b2b0-ed615f42f431/operations",
    "ledgerEventService": "https://bedrock.localhost:18443/ledger-agents/b75ce2a9-bcf0-48e7-b2b0-ed615f42f431/events",
    "ledgerBlockService": "https://bedrock.localhost:18443/ledger-agents/b75ce2a9-bcf0-48e7-b2b0-ed615f42f431/blocks",
    "ledgerQueryService": "https://bedrock.localhost:18443/ledger-agents/b75ce2a9-bcf0-48e7-b2b0-ed615f42f431/query"
  },
  "latestConfigEvent": {...}
}
```

### CODE REVIEW

**Comment by dlongley** [comment]
We'll need to bikeshed `peerId` before we pull this in. Putting `Id` on the end of anything does not follow linked data norms.

File: lib/LedgerAgent.js [hunk 1]
```diff
@@ -11,10 +11,11 @@ const routes = config['ledger-agent'].routes;
 module.exports = class LedgerAgent {
   constructor(options) {
     this.id = options.id;
-    this.node = options.node;
+    this.ledgerNode = options.ledgerNode;
     this.owner = options.owner;
     this.name = options.name;
     this.description = options.description;
+    this.peerId = options.peerId;
```
Comment (dlongley) [inline lib/LedgerAgent.js]: Let's expose this as `targetNode` and change in ledger agent internally. From a "ledger agent perspective" the "peer ID" it gets from consensus should be its `targetNode`.

**Comment by dlongley** [comment]
Result of a mini-bikeshed is to use `targetNode` instead of `peerId` at the ledger agent level (and that's the term that is exposed in the data model).

File: lib/LedgerAgent.js (hunk 1 above)
Comment (mattcollier) [reply lib/LedgerAgent.js]: @dlongley sounds fine.

**Comment by mattcollier** [comment]
@dlongley this is ready for another pass.

**Comment by dlongley** [comment]
Note: We'll need to add `targetNode` to the web ledger context.

**Comment by mattcollier** [comment]
400 TestCloud passes on commit d0afca4.  Merging this.

**Pull request merged**